updated model with changes that came out in implementation (#25)
heritagefund/tech-docs: source/documentation/data_dictionary/tables/_organisation_types.md

> ## organisation types

Reviewer: ```suggestion
## organisation_types
```

> Describes the type of organisation.  Such as TBA

Reviewer: ```suggestion
Describes the type of organisation, e.g. 'local authority'.
```
